cypress/support/readPdf.ts

```diff
@@ -0,0 +1,13 @@
+import fs from "fs";
+import path from "path";
+import pdf from "pdf-parse";
+
+const readPdf = async (pathToPdf: string): Promise<string> => {
+    const pdfPath = path.resolve(pathToPdf);
+    const dataBuffer = fs.readFileSync(pdfPath);
```

Comment:
you can also `await fs.readFile(pdfPath)` although I don't think it makes a difference